mock/docs/site-defaults.cfg

```diff
@@ -498,7 +498,18 @@
 # the whole /etc/pki/ca-trust/extracted directory recursively by default, and
 # Fedora or EL systems work with that.  But some destination chroots can have
 # different configuration, and copying the bundle helps.
-#config_opts['ssl_ca_bundle_path'] = None
+# config_opts['ssl_ca_bundle_path'] = None
```

Comment:
Nit, this is an intentional style (no space).  It helps the reader to more easily tell what
is the documentation text (space after the hash) and what is a commented-out code.

Reply:
Fixed.